ios/LayoutReanimation/REASharedTransitionManager.m

```diff
@@ -15,6 +15,7 @@ @implementation REASharedTransitionManager {
   NSMutableArray<UIView *> *_addedSharedViews;
   BOOL _isSharedTransitionActive;
   NSMutableArray<REASharedElement *> *_sharedElements;
```

Comment:
Does it still make sense to have `_sharedElements` since we now have `_sharedElementsLookup`?

Reply:
Yes, it does. The `_sharedElements` is an array that maintains the views in the correct order to preserve the proper z-index. The `_sharedElementsLookup` is used to speed up the search operation within the shared element set. However, the lookup table is not necessary, and we can remove it by replacing it with a simple iteration over the array. It will be just a bit slower.